src/Illuminate/Mail/Markdown.php

```diff
@@ -69,9 +69,27 @@ public function render($view, array $data = [], $inliner = null)
         $contents = $bladeCompiler->usingEchoFormat(
             'new \Illuminate\Support\EncodedHtmlString(%s)',
             function () use ($view, $data) {
-                return $this->view->replaceNamespace(
-                    'mail', $this->htmlComponentPaths()
-                )->make($view, $data)->render();
+                EncodedHtmlString::encodeUsing(function ($value) {
+                    $replacements = [
+                        '[' => '\[',
+                        '<' => '&lt;',
+                        '>' => '&gt;',
+                    ];
+
+                    $html = str_replace(array_keys($replacements), array_values($replacements), $value);
```

Comment:
Assuming, we could add the following to `EncodedHtmlString`
```php
/**
 * @param  string[]  $chars
 * @param  int-mask-of<HTML_SPECIALCHARS|HTML_ENTITIES>  $table
 * @param  int-mask-of<ENT_*>  $flags
 */
function replaceHtmlEntities(string $str, array $chars, int $table = HTML_SPECIALCHARS, int $flags = ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401, string $encoding = "UTF-8") {
    $map = get_html_translation_table($flags);
    
    return str_replace($chars, array_map(fn ($char) => $map[$char], $chars), $str);
}
```

we could do
```suggestion
                    $html = addcslashes(EncodedHtmlString::replaceHtmlEntities($value, ['<', '>']), '[');
```